RELEASE-NOTES.txt

```diff
@@ -1,3 +1,6 @@
+3.6
```

Comment:
I think you're going to address this next, so no need to hold up the PR with it!